war/.yarnrc.yml

```diff
@@ -1,8 +1,3 @@
-# Defines what linker should be used for installing Node packages (useful to
```

Comment:
any yarn command that modifies this file seems to remove all comments.

in this case I used https://yarnpkg.com/cli/set/version